### DISCUSSION TITLE
Lunii v3

### DISCUSSION THREAD

**Discussion by ex23** [kairoh/studio#125] [General]
Hi,

Does anyone know if STUdio is compatible with the Lunii v3 ?

I have a new Lunii v3, but when I start STUdio and connect the Lunii v3 with a USB cable, I get the following error message.

"Echec d'accès à la fabrique à histoire" (access failed)

And in the console:

2023-10-07 15:00:16,000 ERROR [stu.web.ser.StoryTellerService] (ForkJoinPool.commonPool-worker-1) Failed to plug device: java.util.concurrent.CompletionException: studio.core.v1.exception.StoryTellerException: **Unsupported device metadata format version: 6**
...

Thank you very much for your answers !

**Comment by ongietorri** [comment]
AFAIK no it's not compatible with V3. There is an opened issue here: https://github.com/kairoh/studio/issues/128
Hope someone is working on it...

**Reply by StephaneM007** [reply]
I have the same problem with the v3 !

**Reply by StephaneM007** [reply]
Je viens de réussir avec la version 0.5 : https://github.com/DantSu/studio/releases/tag/0.5.0

**Comment by LoicC04** [comment]
Hi @kairoh ,
Would you need help for v3 compatability ?
I would need a heads up to start. You have any spare time ? 
Thanks

**Comment by mclvfr** [comment]
Hi all
Another forked have been made, that seems to work with Lunii v3 
https://github.com/DantSu/studio 

Sinon il ya des anglophones ou en fait on est tous des francophones utilisant un produit français ? 

Regards
